### ISSUE TITLE
High CPU usage when watching a stream with many other tabs open

### ISSUE THREAD

**Issue by soknife** [LiveTL/LiveTL#167]
Since version 4.x, LiveTL's performance on Firefox gets progressively worse with increased CPU usage the more tabs you have open . If you have something like 300+ tabs (even if 299 of them are unloaded or blank), the browser console gets flooded with the following error with as many repeats as you have tabs. 
```
Unchecked lastError value: Error: Could not establish connection. Receiving end does not exist. 304 background.js:46
    <anonymous> moz-extension://aef650fb-7529-4fff-b7b5-94689f3921d4/js/background.js:46
```
At a certain point the browser starts becoming unusable as the core with the thread reaches 100% usage. I have since reverted to 3.0.12 (testing out the various other versions in between) and no longer have this issue.

**To Reproduce**
Open as many blank tabs as you like, then open a live or archived stream

 - i9-10850k
 - Windows 10 v1909
 - Firefox v87.0b9
 - Clean profile with only this add-on

**Comment by KentoNishi** [comment]
We have a full rewrite underway in a separate repo so although this won't be "patched" in the current version, it should be gone in the next major release

**Comment by KentoNishi** [comment]
Fixed in 06c62e3aa0204d58505708a3d7cf73880e30883e
